viewer/src/views/threejs/cad/shading.ts

```diff
@@ -0,0 +1,37 @@
+/*!
+ * Copyright 2020 Cognite AS
+ */
+
+import { createMaterials, Materials } from './materials';
+
+export interface Shading {
+  materials: Materials;
+  updateNodes: (treeIndices: number[]) => void;
+}
+
+export interface DefaultShadingOptions {
+  color: (treeIndex: number) => number[] | undefined;
+}
+
+export function createDefaultShading(options: DefaultShadingOptions) {
```

Comment:
Nitpick: specify returns type to 1) make it easier to see what the function returns and 2) ensure compile errors happens here, and not in calling code, when we change the Shading-interface in the future.